panels/import_model.py

```diff
@@ -671,4 +827,124 @@ def setup_blender_mat(blender_mat, material_label, ssbh_matl: ssbh_data_py.matl_
+            aim_entry_empty[f'unk{unk_index}'] = aim_entry[f'unk{unk_index}']
+        create_aim_type_helper_bone_constraints(aim_entry['name'], armature, aim_entry['target_bone_name1'], aim_entry['aim_bone_name1'])
+          
+    interpolation_entries_empty = create_new_empty('interpolation_entries', root_empty)
+    for interpolation_entry in nuhlpb_json['data']['Hlpb']['interpolation_entries']:
+        ie = interpolation_entry
+        ie_empty = create_new_empty(ie['name'], interpolation_entries_empty)
+        ie_empty['bone_name'] = ie['bone_name']
+        ie_empty['root_bone_name'] = ie['root_bone_name']
+        ie_empty['parent_bone_name'] = ie['parent_bone_name']
+        ie_empty['driver_bone_name'] = ie['driver_bone_name']
+        ie_empty['unk_type'] = ie['unk_type']
+        aoi = ie['aoi']
+        ie_empty['aoi'] = [aoi['x'], aoi['y'], aoi['z']]
+        quat1 = ie['quat1']
+        ie_empty['quat1'] = [quat1['x'],quat1['y'],quat1['z'],quat1['w']]
+        quat2 = ie['quat2']
+        ie_empty['quat2'] = [quat2['x'],quat2['y'],quat2['z'],quat2['w']]
+        range_min = ie['range_min']
+        ie_empty['range_min'] = [range_min['x'], range_min['y'], range_min['z']]
+        range_max = ie['range_max']
+        ie_empty['range_max'] = [range_max['x'], range_max['y'], range_max['z']]
+        create_interpolation_type_helper_bone_constraints(
+            ie['name'], armature,
+            ie['driver_bone_name'], ie['parent_bone_name'],
+            [aoi['y'], aoi['x'], aoi['z']]
+        )
+    '''
+    list_one and list_two can be inferred from the aim and interpolation entries, so no need to track
+    '''
+
+
+def create_aim_type_helper_bone_constraints(constraint_name, armature, owner_bone_name, target_bone_name):
+    bpy.ops.object.mode_set(mode='POSE', toggle=False)
+    #print(f'{constraint_name}, {armature}, {owner_bone_name}, {target_bone_name}')
+    owner_bone = armature.pose.bones.get(owner_bone_name, None)
+    if owner_bone is not None:
+        new_constraint = owner_bone.constraints.new('DAMPED_TRACK')
+        new_constraint.name = constraint_name
+        new_constraint.track_axis = 'TRACK_Y'
+        new_constraint.influence = 1.0
+        new_constraint.target = armature
+        new_constraint.subtarget = target_bone_name
+    bpy.ops.object.mode_set(mode='OBJECT', toggle=False)
+
+
+def create_interpolation_type_helper_bone_constraints(constraint_name, armature, owner_bone_name, target_bone_name, aoi_xyz_list):
+    bpy.ops.object.mode_set(mode='POSE', toggle=False)
+    owner_bone = armature.pose.bones.get(owner_bone_name, None)
+    if owner_bone is not None:
+        x,y,z = 'X', 'Y', 'Z'
+        for index, axis in enumerate([x,y,z]):
+            crc = owner_bone.constraints.new('COPY_ROTATION')
+            crc.name = f'{constraint_name}.{axis}'
+            crc.target = armature
+            crc.subtarget =  target_bone_name
+            crc.target_space = 'LOCAL_OWNER_ORIENT'
```

Comment:
This breaks compatibility with Blender 2.93. This option was added for 3.0. https://docs.blender.org/api/2.93/bpy.types.Constraint.html#bpy.types.Constraint.target_space

Reply:
oh yeah i think i misunderstood local_owner_orientation anyways. im not really sure how to get consistent interpolation type helper bone entry behavior, going to try to make it work with local space and that will maintain pre 3.0 support.